app/controllers/posts_controller.rb

```diff
@@ -2,4 +2,8 @@ class PostsController < ApplicationController
   def index
     @pagy, @posts = pagy Post.all.includes(:user), limit: Settings.post.limit
```

Comment:
```suggestion
    @pagy, @posts = pagy Post.includes(:user), limit: Settings.post.limit
```
xóa được ko?
cơ mà đã màn list nên có sort có thể theo time hoặc name nhé

Reply:
vâng ở nhánh khác e làm rồi nên pull sau là có luôn ạ.